README.md

```diff
@@ -36,6 +36,45 @@ Install package via PyPI
 pip3 install dxfeed
 ``` 
 
+## Installation from sources
+
+Reminder: initialize and pull git submodule after cloning the repo:
+```bash
+git submodule init
+git submodule update
+``` 
+
+To install dxfeed from source you need Poetry, Cython and taskipy in addition to the normal dependencies above.
+Poetry provides a custom installer. This is the recommended way of installing poetry according to
+[documentation](https://python-poetry.org/docs/)
+
+For osx / linux / bashonwindows:
```

Comment:
`macOS, Linux, Windows (using bash)` 😉 
Also it will be nice to add link to the official project's manual, so if user face any problem with installation of poetry, he'll be able to check it.